src/@utils/aquarius/index.ts

```diff
@@ -41,6 +41,18 @@ export function escapeEsReservedCharacters(value: string): string {
  */
 type TFilterValue = string | number | boolean | number[] | string[]
 type TFilterKey = 'terms' | 'term' | 'match' | 'match_phrase'
+type Query = {
```

Comment:
suggestion: remove in favor of new interface next to `BoolFilter` in `aquarius/index.ts`. Sidenote: `Query` name seems too generic since only one specific subset of query is meant by this.